### ISSUE TITLE
"vendor/acquia/blt/composer.suggested.json" causing issues with composer

### ISSUE THREAD

**Issue by arknoll** [acquia/blt#1325]
My system information:
* Operating system type: OSX
* Operating system version: Sierra
* BLT version: 8.7.1

When I am updating lightning like this:
```
composer require acquia/lightning:^2.1.0
rm -rf vendor && composer install
```
I got all the correct updated dependencies.
Then I ran:
```
composer update
```
and it downgraded me to older versions. For instance:
```
Updating drupal/core (8.3.0 => 8.2.7) Loading from cache
```
Then I ran:
```
composer update
```
again. And nothing was updated.
Then I removed from my composer.json:
```
"vendor/acquia/blt/composer.suggested.json"
```
And ran composer update and I got all the correct dependencies again.

Need to figure out why the suggested json is causing this problem.

**Comment by kylebrowning** [comment]
Same here

**Comment by arknoll** [comment]
More info:
If I leave "vendor/acquia/blt/composer.suggested.json" in and add 
```
{
  "require": {
    "acquia/lightning": "2.1.0",
    "drupal/search_api": "1.0-beta5"
  }
}
```
to blt/composer.overrides.json and run composer update, it still doesn't work.

**Comment by grasmash** [comment]
This seems like one of the packages in composer.suggested.json needs to be changed. Are you sure that the Lightning version is what is causing the problem?

**Comment by arknoll** [comment]
This is what has worked for me:

- Removed "vendor/acquia/blt/composer.suggested.json" from composer file
- Added to composer file:
```
    "require": {
        "drupal/acquia_connector": "^1.5.0",
        "drupal/cog": "^1.0.0",
        "drupal/memcache": "2.x-dev",
        "drupal/seckit": "^1.0.0-alpha2",
        "drupal/security_review": "*",
        "drupal/shield": "^1.0.0"
     },
     "extra": {
        "patches": {
           "drupal/memcache": {
              "SASL Support in Drupal 8 version": "https://www.drupal.org/files/issues/sasl_drupal_8-2862230-3.patch"
            }
         },
```

**Comment by grasmash** [comment]
Best solution is to get this merged: https://github.com/wikimedia/composer-merge-plugin/issues/142

**Comment by justinlevi** [comment]
yeah, I just ran into this issue as well and kept getting the following error until I merged `composer.suggested.json` into the root `composer.json` file.

```
  [Exception]                                                                                                               
  Cannot apply patch SASL Support in Drupal 8 version (https://www.drupal.org/files/issues/sasl_drupal_8-2862230-3.patch)!
```

**Comment by grasmash** [comment]
@justinlevi that is due to #1380, which was resolved in 8.8.2.

**Comment by grasmash** [comment]
Resolved by #1430
